core/src/main/scala/org/broadinstitute/dsde/rawls/model/SamModel.scala

```diff
@@ -99,7 +100,7 @@ object SamResourceTypeAdminActions {
   val readSummaryInformation = SamResourceAction("admin_read_summary_information")
 }
 
-object SamGoogleProjectActions {
+object SamTdrGoogleProjectActions {
```

Comment:
These actions are only checked by the registration APIs so I just renamed this rather than introducing a new object